src/pages/index.js

```diff
@@ -1,4 +1,5 @@
 import * as React from "react"
+import '../fonts/Inter.var.woff2'
```

Comment:
It's necessary to import the font into a JS file so that webpack is aware of it and will bundle it.